Cargo.toml

```diff
@@ -26,6 +26,7 @@ wasmer-cache = { version = "1.0.2", path = "lib/cache", optional = true }
 wasmer-types = { version = "1.0.2", path = "lib/wasmer-types" }
 wasmer-middlewares = { version = "1.0.2", path = "lib/middlewares", optional = true }
 cfg-if = "1.0"
+loupe = { path = "../loupe/crates/loupe" }
```

Comment:
Commenting for completeness: this line needs to change before we can ship this

This applies to all `Cargo.toml` files edited with this change